app/components/marketplace/tag/admin_component/admin_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render CardComponent.new(dom_id: dom_id(tag)) do |card| %>
+  <div class="flex flex-col gap-2">
+    <h3><%= tag.label %></h3>
+    <span class="text-xs">Assigned to <%= assigned_product_count %> <%= "Product".pluralize(assigned_product_count) %></span>
```

Comment:
Not sure about "assigned to" 🤔

Reply:
I think just "N products" or "on N products" might be clear enough.